core/domain/services/graphql/connection_resolvers/FormSectionConnectionResolver.php

```diff
@@ -91,4 +117,25 @@ public function get_query_args(): array
             $this->args
         );
     }
+
+
+    /**
+     * This sets up the "allowed" args, and translates the GraphQL-friendly keys to model
+     * friendly keys.
+     *
+     * @param array $where_args
+     * @return array
+     */
+    public function sanitizeInputFields(array $where_args): array
+    {
+        $arg_mapping = [
+            'appliesTo' => 'FSC_appliesTo',
+            'belongsTo' => 'FSC_belongsTo',
+            'status'    => 'FSC_status',
+        ];
+        return $this->sanitizeWhereArgsForInputFields(
+            $where_args,
+            $arg_mapping
+        );
+    }
```

Comment:
unless we are going to need to manipulate this array in some way, let's just roll it right into the function call and skip using a variable, ie:

```suggestion
    public function sanitizeInputFields(array $where_args): array
    {
        return $this->sanitizeWhereArgsForInputFields(
            $where_args,
            [
            	'appliesTo' => 'FSC_appliesTo',
            	'belongsTo' => 'FSC_belongsTo',
            	'status'    => 'FSC_status',
        	]
        );
    }
```

Reply:
Will make that change in the next PR. Thank you

Reply:
Done [here](https://github.com/eventespresso/event-espresso-core/pull/3441/files#r658090766)

Reply:
🙇🏻